Amazon.IonObjectMapper.Test/NamingConventionTest.cs

```diff
@@ -13,6 +13,9 @@
 
 namespace Amazon.IonObjectMapper.Test
 {
+    using System;
+    using System.IO;
+    using Amazon.IonDotnet.Tree;
```

Comment:
seems we don't need these imports?

Reply:
We need `System` and `System.IO` imports for `Console` and `MemoryStream` respectively, but I've removed the `Amazon.IonDotnet.Tree` import

Reply:
seems we are not using `Console` and `MemoryStream` in `Amazon.IonObjectMapper.Test/NamingConventionTest.cs` and we are not changing the test methods but just add new imports that didn't exist in current file https://github.com/amzn/ion-object-mapper-dotnet/blob/main/Amazon.IonObjectMapper.Test/NamingConventionTest.cs

Reply:
Ah we were referring to different files- I removed those imports in `Amazon.IonObjectMapper.Test/NamingConventionTest.cs`